ecs/cloudformation.go

```diff
@@ -290,15 +295,21 @@ func computeRollingUpdateLimits(service types.ServiceConfig) (int, int, error) {
 	return minPercent, maxPercent, nil
 }
 
-func (b *ecsAPIService) createListener(service types.ServiceConfig, port types.ServicePortConfig,
-	template *cloudformation.Template,
-	targetGroupName string, loadBalancer awsResource, protocol string) string {
+func (b *ecsAPIService) createListener(project *types.Project, service types.ServiceConfig, port types.ServicePortConfig, template *cloudformation.Template, targetGroupName string, loadBalancer awsResource, protocol string) string {
```

Comment:
```suggestion
func (b *ecsAPIService) createListener(project *types.Project, service types.ServiceConfig, port types.ServicePortConfig,
	template *cloudformation.Template, targetGroupName string, loadBalancer awsResource, protocol string) string {
```

nudge to pass `golangci-lint` CI stage